src/store/modules/auth.js

```diff
@@ -110,8 +110,21 @@ export default {
 
     ...withMeta({
       async save ({ dispatch }, data) {
-        const savedUser = await dispatch('backgroundSave', data)
-        router.push({ name: 'user', params: { userId: savedUser.id } })
+        try {
+          await dispatch('backgroundSave', data)
+          dispatch('toasts/show', {
+            message: 'Your changes have been saved!',
```

Comment:
The only thing left would be to make these messages translatable. We use `vue-i18n` for this. You can import our instance from `@/i18n` and use the `i18n.t` function. The messages are defined in `locales/en.json`.